packages/commonwealth/client/scripts/models/Group.ts

```diff
@@ -0,0 +1,57 @@
+interface APIResponseFormat {
+  id: number;
+  chain_id: string;
+  metadata: {
+    name: string;
+    description?: string;
+  };
+  memberships: any[];
+  requirements: {
+    rule: 'threshold';
+    source: {
+      source_type: 'erc20' | 'erc721' | 'cosmos_native' | 'eth_native';
+      evm_chain_id?: number;
+      cosmos_chain_id?: number;
+      contract_address?: string;
+      token_symbol?: string;
+    };
+  }[];
+  topicIds: number[];
+  updated_at: string;
+  created_at: string;
+}
+
+class Group {
+  public id: number;
+  public chainId: string;
```

Comment:
This should be `communityId` since it is associated with the `id` column of the `Chains` table. This change can either be done here or completed later as part of #5356 in a separate PR (doing it later may be easier). Thanks @masvelio for being so attentive and pointing this out!

Reply:
Updated